src/window.rs

```diff
@@ -682,6 +682,28 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused.
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **iOS / Android / Web / Wayland:** Unsupported.
+    ///
+    /// - **MacOS:** `RequestUserAttentionType::Critical` bounces the dock icon until the
+    /// application is in focus. `RequestUserAttentionType::Informational` bounces the   
+    /// dock icon once.
+    ///
+    /// - **Windows:** `RequestUserAttentionType::Critical` flashes both the window and the
+    /// taskbar button until the application is in focus. `RequestUserAttentionType::Informational`
+    /// flashes the taskbar button until the application is in focus.
+    ///
+    /// - **X11:** `RequestUserAttentionType::Critical` and `RequestUserAttentionType::Informational`
+    /// provide the exact same result, which is determined by the WM.
```

Comment:
We should move enum variants description to the corresponding enum variants , and not list it on the method.

Reply:
Moved it all to the enum variants of `RequestUserAttentionType` :)